examples/apex_rest_api_usecases/create_case/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org = "salsforce_exsamples"
```

Comment:
```suggestion
org = "salsforce_examples"
```

Reply:
updated all instances in [bbda6d2](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/322/commits/bbda6d24f4c2c825f38ade4a9c7ec2467a42e73d)

Reply:
Updated in [bbda6d2](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/322/commits/bbda6d24f4c2c825f38ade4a9c7ec2467a42e73d)